src/gui/guiFormSpecMenu.cpp

```diff
@@ -1362,14 +1439,21 @@ void GUIFormSpecMenu::parseImageButton(parserData* data, const std::string &elem
 			Environment->setFocus(e);
 		}
 
-		e->setUseAlphaChannel(true);
+		auto style = getStyleForElement("image_button", spec.fname);
+
+		e->setUseAlphaChannel(style.getBool(StyleSpec::ALPHA, true));
 		e->setImage(guiScalingImageButton(
 			Environment->getVideoDriver(), texture, geom.X, geom.Y));
 		e->setPressedImage(guiScalingImageButton(
 			Environment->getVideoDriver(), pressed_texture, geom.X, geom.Y));
 		e->setScaleImage(true);
-		e->setNotClipped(noclip);
-		e->setDrawBorder(drawborder);
+		if (parts.size() >= 7) {
+			e->setNotClipped(noclip);
+			e->setDrawBorder(drawborder);
+		} else {
+			e->setNotClipped(style.getBool(StyleSpec::NOCLIP, false));
+			e->setDrawBorder(style.getBool(StyleSpec::BORDER, false));
```

Comment:
`drawborder` default to `true` if not specified.

Reply:
nice catch